org.flameshot.Flameshot.yml

```diff
@@ -0,0 +1,33 @@
+app-id: org.flameshot.Flameshot
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: flameshot
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  - --device=dri
+  # Connectivity
+  - --share=network
+  # QtSingleApplication, allow other instances to see log files
+  - --env=TMPDIR=/var/tmp
+  # Allow loading/saving files from anywhere
+  - --filesystem=host
+  # Notification access
+  - --talk-name=org.freedesktop.Notifications
+  # System Tray Icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-1
```

Comment:
This happens to work only if you don't have other flatpaked Qt application running as applications almost always run as PID 2. You'll need `--own-name=org.kde.*` here.